cli/src/main.rs

```diff
@@ -1,5 +1,6 @@
 #[macro_use]
 extern crate clap;
+mod package;
```

Comment:
does the ordering here matters? can you put Mod before extern? does it affect anything?

Reply:
it doesn't, I think we should add linters and use a style that seems the leading one